LICENSE

```diff
@@ -1,3 +1,8 @@
+All files except for the following are covered by the MIT license:
+
+- files in src/shared/third-party/
+- files in src/third-party/
+
```

Comment:
I'd prefer a slightly more vague wording, since we currently still have differently licensed content outside of `third-party` folders that is individually called out in the source code.
```suggestion
Unless otherwise noted, most files and code are licensed under the MIT license.
```